canonical-web-and-design/juju.is
# Build the about devsecops page

## QA
- Go to http://0.0.0.0:8041/about/devsecops
- Check the content matches the [copy doc](https://docs.google.com/document/d/10tjWftKVRrmsUn9R5vSLlRtLS0o0nwBbKA4Av5m840w/edit#)

Fixes https://github.com/canonical-web-and-design/juju.is/issues/100
